lib/tasks/tag_tasks.rake

```diff
@@ -49,6 +49,23 @@ namespace :Tag do
     end
   end
 
+  desc "Reset non canonical warnings"
+  task(reset_non_canonical_warnings: :environment) do
+    choose_not_to_warn = Warning.find_by(name: "Choose Not To Use Archive Warnings")
+    Warning.where(canonical: [false, nil]).each do |warning|
+      if warning.taggings_count.zero?
```

Comment:
The tag deletion task also checks for child tags and inclusion in a tag set:
https://github.com/otwcode/otwarchive/blob/c8fe1392b0493dfd5730be590d3b60715414791e/lib/tasks/tag_tasks.rake#L40-L45
Do those need to be checked here, as well? (Or, to avoid code duplication, would it be worth getting rid of the call to `warning.destroy` here and just calling the tag deletion task afterwards?)

Reply:
I think calling the tag clean up afterwards makes sense.